web/src/site-admin/backend.tsx

```diff
@@ -304,6 +305,118 @@ export function fetchSite(): Observable<GQL.ISite> {
     )
 }
 
+type SettingsConfig = Pick<GQL.SettingsSubject, 'settingsURL' | '__typename'> & {
+    contents: any
+}
+
+/**
+ * All configuration and settings in one place. Excludes critical site config for now, because this
+ * must be fetched from management console
+ */
+export interface AllConfig {
+    site: any
+    externalServices: Partial<
```

Comment:
```suggestion
    externalServices: Partial<Record<GQL.ExternalServiceKind, any>>
```

Is there a way to avoid `any` here?